Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.py

```diff
@@ -0,0 +1,832 @@
+        'rawJSON': json.dumps(incident)
+    } for incident in incidents]
+
+    next_run = {
+        'last_fetch_time': incidents[-1].get('date_opened') if incidents else minimum_date_opened_str
+    }
+    return incidents_obj_list, next_run
+
+
+def test_module(client: Client, params: Dict) -> str:
+    """
+    Tests API connectivity and authentication.
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    Args:
+        client (Client): FraudWatch client to perform the API call.
+        params (Dict): Demisto params.
+
+    Returns:
+        (str): 'ok' if test passed, anything else will fail the test.
+    """
+    try:
+        fetch_incidents_command(client, params, {})
+        return 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+            e = DemistoException(message)
+        raise e
+
+
+def fraud_watch_incidents_list_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Gets a list of incidents from FraudWatch service with the possible filters:
+    - Brand: Retrieve incidents which corresponds to the given brand. throws Exception if brand does not exist.
+    - Status: Retrieve incidents which corresponds to the given status. Unknown status will return empty incident list.
+    - Limit: Total number of Incidents in a page. The default limit is 20 and the maximum number is 200.
+    - Page: Retrieve incidents by the given page number.
+    - From Date: Retrieve alerts that their date opened is higher or equal to 'from' value.
+                 Supports ISO and time range (<number> <time unit>, e.g., 12 hours, 7 days) formats.
+                 If 'to' argument is not given, default value for 'to' is current day.
+    - To Date: Retrieve alerts that their date opened is lower or equal to 'to' value.
+               Supports ISO and time range (<number> <time unit>, e.g., 12 hours, 7 days) formats.
+               If 'from' argument is not given, default value for 'from' is 12 months before 'to'.
+
+    Known errors that causes error to be returned by FraudWatch service:
+    - Unknown brand.
+    - Invalid 'to_date' or 'from_date' values.
+
+    Args:
+        client (Client): FraudWatch client to perform the API calls.
+        args (Dict): Demisto arguments.
+
+    Returns:
+        CommandResults.
+    """
+    brand = args.get('brand')
+    status = args.get('status')
+    page = get_and_validate_positive_int_argument(args, 'page')
+    limit = get_and_validate_positive_int_argument(args, 'limit')
+
+    from_date = get_time_parameter(args.get('from'))
+    if from_date:
+        from_date = from_date.strftime('%Y-%m-%d')
+
+    to_date = get_time_parameter(args.get('to'))
+    if to_date:
+        to_date = to_date.strftime('%Y-%m-%d')
```

Comment:
Nice fature of python 3.8
```suggestion
    if from_date := get_time_parameter(args.get('from'))
        from_date = from_date.strftime('%Y-%m-%d')

    if to_date := get_time_parameter(args.get('to'))
        to_date = to_date.strftime('%Y-%m-%d')
```

Reply:
Cool!